module/userProjects/helper.js

```diff
@@ -1030,6 +1083,118 @@ module.exports = class UserProjectsHelper {
         })
     }
 
+
+      /**
+   * Implement find query for entity
+   * @method
+   * @name entityDocuments
+   * @param {Object} [findQuery = "all"] - filter query object if not provide 
+   * it will load all the document.
+   * @param {Array} [fields = "all"] - All the projected field. If not provided
+   * returns all the field
+   * @param {Number} [limitingValue = ""] - total data to limit.
+   * @param {Number} [skippingValue = ""] - total data to skip.
+   * @returns {Array} - returns an array of entities data.
+   */
+
+      static entityDocuments(findQuery = "all", fields = "all", limitingValue = "", skippingValue = "",sortedData = "") {
```

Comment:
@Prajwal17Tunerlabs you dont need to query from entities collection. You need to call entity service where ever you required entity information